Project.toml

```diff
@@ -0,0 +1,20 @@
+name = "PowerSystemsUnits"
+uuid = "c279aad7-7dc9-58d3-995b-109d3f5deb94"
+license = "MIT"
+author = "Invenia Technical Computing"
+version = "0.2.0"
+
+[deps]
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
+
+[compat]
+Unitful = "0.14, 0.15"
+julia = "~1.0.4, 1.1.1"
```

Comment:
Is this package broken on 1.0.3 and earlier and on 1.1.0?

Reply:
The changes I made (around __init__ and precompilation) require https://github.com/JuliaLang/julia/pull/31047

The older version of the package should still work on older versions of Julia